docs/source/getting_started/index.rst

```diff
@@ -0,0 +1,112 @@
+Getting started
+===============
+
+In this section, we will create an environment to run Iroha instance, play around with settings a bit, launch it, create a couple of transactions, and check the data written in the ledger. Also, we will deploy a network of peers and will try to emulate rainy day scenarios happening in the network.
+
+.. note:: Ledger is the synonym for a blockchain, and Hyperledger Iroha is known also as Distributed Ledger Technology — which in essence is the same as "blockchain framework". You can check the rest of terminology used in the Glossary section.
+
+Let's prepare the environment
+-----------------------------
+
+.. hint:: Having troubles? Check FAQ section or communicate to us directly, in case you were stuck on something. We don't expect this to happen, but some issues with an environment are possible.
+
+Prerequisite 1
+^^^^^^^^^^^^^^
+
+Prerequisite 2
+^^^^^^^^^^^^^^
+
+Prerequisite 3
+^^^^^^^^^^^^^^
+
+Getting Iroha build
+-------------------
+
+
+Step 1
+^^^^^^
+
+Step 2
+^^^^^^
+
+Step 3
+^^^^^^
+
+Playing around with the settings
+--------------------------------
+
+
+Step 1
+^^^^^^
+
+Step 2
+^^^^^^
+
+Step 3
+^^^^^^
+
+Playing around with settings
+----------------------------
+
+
+Step 1
+^^^^^^
+
+Step 2
+^^^^^^
+
+Step 3
+^^^^^^
+
+Running a single instance
+-------------------------
+
+
+Step 1
+^^^^^^
+
+Step 2
+^^^^^^
+
+Step 3
+^^^^^^
+
+Sending transactions and queries
+--------------------------------
+
+
+Step 1
+^^^^^^
+
+Step 2
+^^^^^^
+
+Step 3
+^^^^^^
+
+Deploy a network of Iroha peers
+-------------------------------
+
+
+Step 1
+^^^^^^
+
+Step 2
+^^^^^^
+
+Step 3
+^^^^^^
+
+
+What if bad things will happen?
```

Comment:
Here we can give some only top "bad things" that can happen? Maybe we can also make a separation section to discuss them

Reply:
This is just a proposed topic like what if f peers will fail, what if they send bad data, what if the network is down, etc. The purpose of this section is just to boost confidence by showing what we really are capable of when faults happen

Reply:
Let's maybe put something like fault-tolerance section also in core concepts

Reply:
@grimadas what do you think

Reply:
Ouugh, yeah, then core concepts sounds nice.